services/api/schemas/samples.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel
+
+
+class Sample(BaseModel):
+    # Does this take anything by default..?
+    pass
+
+
+class FileSample(Sample):
+    file_name: str
+
+
+class ShotSample(Sample):
+    shot_id: int
+
+
+class UFOSample(Sample):
```

Comment:
I think this might be too specific.

I am thinking we have roughly two types of `Samples`:

 - Samples where they are names of files (e.g. CSV, png, parquet, mp4).
 - Samples where they index a shot number. (e.g. for UDA, SAL)

Additionally, I am wondering if we really need both a camera and a frame for UFOs? Wouldn't it make more sense to have a whole video, with all the annotated frames, and the video viewer just jumps to the first UFO?